src/net.h

```diff
@@ -1215,7 +1263,7 @@ class CNode
 
     // flood relay
     std::vector<CAddress> vAddrToSend;
-    std::unique_ptr<CRollingBloomFilter> m_addr_known = nullptr;
+    std::unique_ptr<CRollingBloomFilter> m_addr_known{nullptr};
```

Comment:
(see net.h:1274)
we still have `IsAddrRelay` usages over code base.
Please, drop remaining usages and this method; update related comment also:
```diff
-// in dash: m_tx_relay should never be nullptr, use `IsAddrRelayPeer() == false` instead
+// in dash: m_tx_relay should never be nullptr, use `RelayAddrsWithConn() == false` instead
```

Reply:
got it